app/components/course-admin/tester-versions-page/version-list-item.hbs

```diff
@@ -29,6 +29,20 @@
           {{date-from-now @courseTesterVersion.createdAt}}
         </p>
       {{/if}}
+
+      {{#if (gt @courseTesterVersion.provisionedTestRunnersCount 0)}}
+        <div class="flex items-center gap-x-2">
+          <svg viewBox="0 0 2 2" class="h-0.5 w-0.5 fill-current">
```

Comment:
The icon needs to be green too here btw, not sure if that's the case at the moment